### PR TITLE
Feat: loading bar

### PR DESCRIPTION
**Pull request by SamCreasey**
Added the [Loading bar](https://www.figma.com/file/pevuvey1joCsopTRr9f4OE/Fondue---Components?node-id=12398%3A207074) component .

The component does not emit the events described in the Figma component as we decided it was no longer necessary

### CODE REVIEW

File: src/components/LoadingBar/LoadingBar.tsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { merge } from '@utilities/merge';
+import React from 'react';
```
Comment (dominiquemb) [inline src/components/LoadingBar/LoadingBar.tsx:4]: The only thing I would change here is move the 'react' import to the top.

File: src/components/LoadingBar/LoadingBar.cy.tsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import {
+    LoadingBar,
+    LoadingBarSize,
+    LoadingBarState,
+    loadingBarSizeClasses,
+    loadingBarStateClasses,
+} from './LoadingBar';
+import React from 'react';
```
Comment (dominiquemb) [inline src/components/LoadingBar/LoadingBar.cy.tsx:10]: Move the 'react' import to the top of the file.

File: src/components/LoadingBar/LoadingBar.tsx [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+};
+
+export const LoadingBar = ({
+    percentage = 0,
+    size = LoadingBarSize.Medium,
+    state = LoadingBarState.Progress,
+    rounded = false,
+}: LoadingBarProps) => (
+    <div
+        className={merge([
```
Comment (dominiquemb) [inline src/components/LoadingBar/LoadingBar.tsx:55]: Use `joinClassNames` here.

File: src/components/LoadingBar/LoadingBar.tsx [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+            'tw-overflow-hidden tw-w-full tw-relative',
+            loadingBarSizeClasses[size],
+            rounded && 'tw-rounded',
+            loadingBarStateClasses[state].outer,
+        ])}
+        data-test-id="loading-bar-outer"
+    >
+        <div
+            className={merge([
```
Comment (dominiquemb) [inline src/components/LoadingBar/LoadingBar.tsx:64]: Use `joinClassNames` here.

**Review by dominiquemb — CHANGES_REQUESTED** [review]

File: src/components/LoadingBar/LoadingBar.tsx (hunk 1 above)
Comment (SamCreasey) [reply src/components/LoadingBar/LoadingBar.tsx:3]: fixed

File: src/components/LoadingBar/LoadingBar.cy.tsx (hunk 2 above)
Comment (SamCreasey) [reply src/components/LoadingBar/LoadingBar.cy.tsx]: fixed

File: src/components/LoadingBar/LoadingBar.tsx (hunk 4 above)
Comment (SamCreasey) [reply src/components/LoadingBar/LoadingBar.tsx:64]: Not available in Fondue

File: src/components/LoadingBar/LoadingBar.tsx (hunk 3 above)
Comment (SamCreasey) [reply src/components/LoadingBar/LoadingBar.tsx:55]: Not available in Fondue

**Review by dominiquemb — APPROVED** [review]

**Review by dominiquemb — APPROVED** [review]

**Pull request merged**